variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/AbstractVariant.java

```diff
@@ -181,11 +176,23 @@ public String getMainId() {
         return mainId;
     }
 
-    public void renormalizeVariant(long newStart, long newEnd, String newReference, String newAlternate) {
-        this.start = newStart;
-        this.end = newEnd;
-        this.reference = newReference;
-        this.alternate = newAlternate;
+    public void setReference(String reference) {
```

Comment:
We should probably prefer `setReference` and `setAlternate` to be private (or protected if used by a child class, but I assume that is not the case right now).